docs/virtual_machines/tekton_tasks.md

```diff
@@ -130,3 +103,36 @@ for Windows versions, which uses BIOS. More informations about pipeline can be f
 - [Windows efi installer](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-efi-installer-pipeline.yaml) - Pipeline will prepare a template and Windows 11/2k22 datavolume vith virtio drivers installed. User has to provide a link to working Windows 11/2k22 iso file. Pipeline is suitable for Windows versions, which requires EFI (e.g. Windows 11/2k22). More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-efi-installer)
 
 - [Windows customize](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-customize-pipeline.yaml) - Pipeline will install SQL server or VS Code in Windows VM. More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-customize)
+
+### Using tasks and example pipelines in different namespace
+The user can set in which namespace example pipelines or tasks will be deployed by setting `spec.tektonPipelinesNamespace` or `spec.tektonTasksNamespace`in HCO CR:
+
+```console
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: kubevirt-hyperconverged
+spec:
+  tektonPipelinesNamespace: userNamespace
+  tektonTasksNamespace: userNamespace
+```
+
+or in SSP CR by setting `spec.tektonPipelines.namespace` or `spec.tektonTasks.namespace`:
+```console
+apiVersion: ssp.kubevirt.io/v1beta2
+kind: SSP
+metadata:
+  name: ssp
+  namespace: kubevirt
+spec:
+  tektonPipelines:
+    namespace: kubevirt
+  tektonTasks:
+    namespace: kubevirt
+```
+
+The namespace has to exists before doing this change.
```

Comment:
These lines all sound important and in the line of a 'note' admonition, but these lines all currently render as one large para. Would it make sense to separate these with bullet points or similar?

Reply:
I added note tag